templates/bot/ts/default/package.json

```diff
@@ -24,6 +24,7 @@
     "restify": "^8.5.1"
   },
   "devDependencies": {
+    "@types/node": "^12.0.0",
```

Comment:
why this dev dependencies is introduced?

Reply:
Before `npm install`, I meet with below lint warning:
![image](https://user-images.githubusercontent.com/1050213/194739915-0801ba78-411e-47e4-94da-7c0e5c5a5db4.png)
But after `npm install`, I find `@types/node` is already in descendant dev dependencies, so no need to explicitly include it now. Thanks for pointing out. I have removed it.